### PR TITLE
Tweaks to README.md

### LINKED ISSUE #18: Testing feedback
Ran through the instructions creating [ns-rse/workbench-test](https://github.com/ns-rse/workbench-test) as I was checking the instructions for [RSE-Sheffield/uos-varnish](https://github.com/RSE-Sheffield/uos-varnish) so figured I should run through these too.

I realise a lot of this is copy and pasted from the Carpentries Workbench instructions and I'll look to contribute patches up-stream but this is now material that is divergent and considerably different so noting these here.

## `fair4rs_config.patch`

This applied perfectly :+1: . Some suggestions on fields below.

### Edit configuration, citation metadata, contributing guide and licence

Instructions are clear but I was wondering if we could tweak the Markdown 

Perhaps include line numbers in the bulleted points.

### `config.yaml`

```markdown
This file contains global parameters for your lesson site. Individual fields within the file are documented with comments (beginning with `#`). At a minimum, you should adjust all the fields marked `# FIXME`:
```

+ The `created` date has `# FIXME` appended to the line suggesting it needs modifying but it isn't listed in this section of the `README.md`. Should we ask people to enter the date the lesson template is first used and work begins or to leave until material development is ready to be used?
+ `url` is listed but no `# FIXME` at the end of the line. Maybe we could perhaps be helpful and instruct people on what the URL is likely to be based on what account/organisation the repository will be under, if its `RSE-Sheffield` then it will resolve to `https://rse.shef.ac.uk/<repository>`.

### `CITATION.cff`

```markdown
This file contains information about citing this repository. At a minimum, you should adjust all the fields marked `# FIXME`:
```
 
+ Could we perhaps include `university of sheffield` as a keyword?

### `CONTRIBUTING.md`

Would it be helpful to be consistent and use `# FIXME` in this file? On first viewing I struggled to see these and having them stand out might help. If so then...

```markdown
Update links to GitHub repository and issues, marked `<!-- # FIXME -->`.
```
...and we would need to remind people to remove these from the Markdown as they wouldn't be ignored.

Also we know these are likely to be hosted on GitHub so the patch should probably have...

```markdown
[repo]: https://github.com/<account>/<repo>                 <!-- # FIXME -->
[repo-issues]: https://github.com/<account>/<repo>/issues   <!-- # FIXME -->
```

### Commit changes to configuration, citation metadata, contributing guide and licence

Could use the following instead, its more succinct for people to type and adds all tracked files that have been touched/updated.

```bash
git add -u
```

### Activate GitHub Pages for your repository

I know this is a copy and paste (and I will at some point contribute this up-stream) but it would perhaps be useful to mention that under _Settings > Pages_ that the _Source_ field should be  set to _Deploy from branch_ prior to stating what that branch should be.

#### Update README

I think this section would perahps fit beetter under `Edit configuration, citation metadata, contributing guide and licence` so that it is included in the first commit people make. Perhaps to shorten the section title it could _Update Files with Metadata_.

### PR DESCRIPTION
**Pull request by ns-rse**
Closes #18

Hopefully addresses most of the minor points noted in #18 and with a bit of luck I won't have messed up the `.patch`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,32 +15,32 @@ Workbench](https://carpentries.github.io/sandpaper-docs/) provides a
...
 
 Required software:
 
-- git (>= 2.28 recommended)
-- R (>= 3.6)
-- pandoc (>= 2.11) (available through the [RStudio
+- [git][git] (>= 2.28 recommended)
+- [R][r] (>= 3.6)
+- [pandoc][pandoc] (>= 2.11) (available through the [RStudio
   IDE](https://posit.co/download/rstudio-desktop/#download))
```
Comment (tdjames1) [inline README.md:34]:
```suggestion
- [pandoc][pandoc] (>= 2.11) (available through the [RStudio IDE][rstudio])
```

**Review by tdjames1 — CHANGES_REQUESTED** [review]
Thanks, Neil, I've suggested a change to the tweak to CONTRIBUTING.md and I spotted another link that could be shortened.

File: fair4rs_config.patch [hunk 2]
```diff
@@ -143,10 +144,12 @@ index 6c2b81c..f71bc09 100644
...
 -email][contact].
- 
- [repo]: https://example.com/FIXME
- [repo-issues]: https://example.com/FIXME/issues
+
+-[repo]: https://example.com/FIXME
+-[repo-issues]: https://example.com/FIXME/issues
 -[contact]: mailto:team@carpentries.org
++[repo]: https://gibhub.com/<account>/<repo>                 <!-- # FIXME -->
++[repo-issues]: https://github.com/<account>/<repo>/issues   <!-- # FIXME -->
```
[the remark this answers is not in the document]
Comment (tdjames1) [reply fair4rs_config.patch:152]: Also just noticed the [repo] link has a typo!

```suggestion
+[repo]: https://github.com/<account>/<repo>#FIXME
+[repo-issues]: https://github.com/<account>/<repo>/issues#FIXME
```

[the remark this answers is not in the document]
Comment (ns-rse) [reply fair4rs_config.patch:152]: Ah, missed that this doesn't render, thanks for correcting and spotting the typo.

File: README.md [hunk 3]
```diff
@@ -134,17 +137,22 @@ minimum, you should adjust all the fields marked 'FIXME':
 
 ### `CONTRIBUTING.md`
 
-Update links to GitHub repository and issues, marked 'FIXME'.
+Update links to GitHub repository and issues, marked `<!-- # FIXME -->`.
```
Comment (tdjames1) [inline README.md:140]:
```suggestion
Update links to GitHub repository and issues, marked `#FIXME`.
```

**Review by tdjames1 — CHANGES_REQUESTED** [review]
One more change required to reflect change in the patch to CONTRIBUTING.md.

File: README.md (hunk 3 above)
Comment (ns-rse) [reply README.md]: Good catch, thanks.

**Review by tdjames1 — APPROVED** [review]
All good, thanks!

**Pull request merged**